WordPress/Classes/ViewRelated/Stats/Shared Views/Stats Detail/SiteStatsDetailTableViewController.swift

```diff
@@ -0,0 +1,142 @@
+import UIKit
+
+class SiteStatsDetailTableViewController: UITableViewController, StoryboardLoadable {
+
+    // MARK: - StoryboardLoadable Protocol
+
+    static var defaultStoryboardName = "SiteStatsDetailTableViewController"
```

Comment:
Based on how the `StoryboardLoadable` protocol extension works, it should be possible to use the following if you are so inclined:
```
static var defaultStoryboardName = defaultControllerID
```

Reply:
Oooh, I never noticed that. And it does in fact work. Thanks!